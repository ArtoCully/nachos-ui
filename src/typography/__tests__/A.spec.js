import 'react-native'
import React from 'react'
import renderer from 'react-test-renderer'

import A from '../A'

it('should render correctly', () => {
  const component = renderer.create(
    <A href='http://avocode.com'>This is a link</A>
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
