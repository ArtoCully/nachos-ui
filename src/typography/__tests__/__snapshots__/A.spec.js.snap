exports[`test should render correctly 1`] = `
<Text
  accessibilityRole="link"
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  onPress={[Function]}
  style={
    Array [
      Object {
        "color": "#333",
        "fontSize": 14,
        "fontStyle": "normal",
        "fontWeight": "normal",
        "paddingVertical": 10,
        "textAlign": "left",
      },
      Array [
        Object {
          "color": "#2f8cff",
          "textDecorationColor": "#2f8cff",
          "textDecorationLine": "underline",
          "textDecorationStyle": "solid",
        },
        undefined,
      ],
    ]
  }>
  This is a link
</Text>
`;
